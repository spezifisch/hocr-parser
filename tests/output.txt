|

In facsimilea photocell is caused
to perform a raster scan over the
subject copy. The variations of print
density on the document cause the
photocell to generate an analogous
electrical video signal. This signal is
used to modulate a carrier, which is
transmitted to a remote destination
over a radio or cable
communications link.

At the remote terminal.
demodulation reconstructs the video
signal, which is used to modulate the
density ofprint produced by a
printing device. This device is
scanning in a raster scan
synchronised with that at the
transmitting terminal. As a result, a
facsimile copy of the subject
document is produced.

@ Wei/PUP com

