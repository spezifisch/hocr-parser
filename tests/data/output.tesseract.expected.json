{
    "coordinates": {
        "document": [
            0,
            0,
            0,
            0
        ],
        "block_1_1": [
            83,
            68,
            449,
            376
        ],
        "block_1_2": [
            84,
            415,
            458,
            696
        ],
        "block_1_3": [
            426,
            754,
            536,
            769
        ],
        "line_1_1": [
            83,
            68,
            86,
            96
        ],
        "line_1_10": [
            84,
            329,
            292,
            347
        ],
        "line_1_11": [
            85,
            358,
            300,
            376
        ],
        "line_1_12": [
            117,
            415,
            353,
            434
        ],
        "line_1_13": [
            84,
            443,
            458,
            462
        ],
        "line_1_14": [
            84,
            472,
            456,
            496
        ],
        "line_1_15": [
            84,
            501,
            389,
            524
        ],
        "line_1_16": [
            84,
            529,
            383,
            553
        ],
        "line_1_17": [
            84,
            559,
            334,
            581
        ],
        "line_1_18": [
            84,
            586,
            380,
            610
        ],
        "line_1_19": [
            84,
            615,
            442,
            638
        ],
        "line_1_2": [
            118,
            101,
            448,
            124
        ],
        "line_1_20": [
            85,
            643,
            371,
            667
        ],
        "line_1_21": [
            84,
            672,
            316,
            696
        ],
        "line_1_22": [
            426,
            754,
            536,
            769
        ],
        "line_1_3": [
            84,
            129,
            416,
            153
        ],
        "line_1_4": [
            84,
            158,
            449,
            182
        ],
        "line_1_5": [
            84,
            186,
            435,
            210
        ],
        "line_1_6": [
            84,
            215,
            439,
            239
        ],
        "line_1_7": [
            85,
            244,
            444,
            267
        ],
        "line_1_8": [
            84,
            272,
            444,
            292
        ],
        "line_1_9": [
            84,
            301,
            446,
            319
        ],
        "page_1": [
            0,
            0,
            545,
            771
        ],
        "par_1_1": [
            83,
            68,
            86,
            96
        ],
        "par_1_2": [
            84,
            101,
            449,
            376
        ],
        "par_1_3": [
            84,
            415,
            458,
            696
        ],
        "par_1_4": [
            426,
            754,
            536,
            769
        ],
        "word_1_1": [
            83,
            68,
            86,
            96
        ],
        "word_1_10": [
            216,
            133,
            277,
            147
        ],
        "word_1_100": [
            259,
            643,
            290,
            661
        ],
        "word_1_101": [
            298,
            643,
            371,
            667
        ],
        "word_1_102": [
            84,
            672,
            187,
            690
        ],
        "word_1_103": [
            192,
            673,
            207,
            690
        ],
        "word_1_104": [
            214,
            672,
            316,
            696
        ],
        "word_1_105": [
            426,
            754,
            438,
            766
        ],
        "word_1_106": [
            443,
            754,
            502,
            769
        ],
        "word_1_107": [
            508,
            757,
            536,
            766
        ],
        "word_1_11": [
            283,
            136,
            328,
            147
        ],
        "word_1_12": [
            334,
            136,
            380,
            147
        ],
        "word_1_13": [
            385,
            129,
            416,
            147
        ],
        "word_1_14": [
            84,
            158,
            158,
            181
        ],
        "word_1_15": [
            162,
            164,
            215,
            182
        ],
        "word_1_16": [
            220,
            158,
            258,
            176
        ],
        "word_1_17": [
            265,
            159,
            368,
            176
        ],
        "word_1_18": [
            374,
            158,
            395,
            176
        ],
        "word_1_19": [
            399,
            159,
            449,
            182
        ],
        "word_1_2": [
            118,
            102,
            137,
            119
        ],
        "word_1_20": [
            84,
            186,
            158,
            210
        ],
        "word_1_21": [
            164,
            193,
            190,
            205
        ],
        "word_1_22": [
            195,
            186,
            226,
            205
        ],
        "word_1_23": [
            234,
            186,
            337,
            205
        ],
        "word_1_24": [
            342,
            193,
            398,
            205
        ],
        "word_1_25": [
            404,
            186,
            435,
            205
        ],
        "word_1_26": [
            84,
            215,
            179,
            239
        ],
        "word_1_27": [
            184,
            219,
            203,
            233
        ],
        "word_1_28": [
            211,
            219,
            300,
            239
        ],
        "word_1_29": [
            305,
            222,
            329,
            233
        ],
        "word_1_3": [
            144,
            101,
            249,
            119
        ],
        "word_1_30": [
            335,
            215,
            439,
            239
        ],
        "word_1_31": [
            85,
            244,
            178,
            262
        ],
        "word_1_32": [
            183,
            244,
            238,
            262
        ],
        "word_1_33": [
            245,
            244,
            308,
            267
        ],
        "word_1_34": [
            314,
            244,
            357,
            262
        ],
        "word_1_35": [
            364,
            244,
            422,
            267
        ],
        "word_1_36": [
            430,
            245,
            444,
            262
        ],
        "word_1_37": [
            84,
            272,
            131,
            290
        ],
        "word_1_38": [
            138,
            276,
            157,
            290
        ],
        "word_1_39": [
            163,
            272,
            259,
            290
        ],
        "word_1_4": [
            257,
            101,
            352,
            124
        ],
        "word_1_40": [
            266,
            279,
            276,
            290
        ],
        "word_1_41": [
            282,
            273,
            357,
            292
        ],
        "word_1_42": [
            362,
            272,
            424,
            290
        ],
        "word_1_43": [
            429,
            273,
            444,
            290
        ],
        "word_1_44": [
            84,
            301,
            203,
            319
        ],
        "word_1_45": [
            209,
            304,
            228,
            319
        ],
        "word_1_46": [
            236,
            307,
            246,
            319
        ],
        "word_1_47": [
            253,
            304,
            325,
            319
        ],
        "word_1_48": [
            332,
            301,
            446,
            319
        ],
        "word_1_49": [
            84,
            336,
            131,
            347
        ],
        "word_1_5": [
            357,
            102,
            372,
            119
        ],
        "word_1_50": [
            136,
            336,
            146,
            347
        ],
        "word_1_51": [
            153,
            329,
            205,
            347
        ],
        "word_1_52": [
            212,
            336,
            234,
            347
        ],
        "word_1_53": [
            240,
            329,
            292,
            347
        ],
        "word_1_54": [
            85,
            359,
            253,
            376
        ],
        "word_1_55": [
            258,
            358,
            300,
            376
        ],
        "word_1_56": [
            117,
            416,
            140,
            433
        ],
        "word_1_57": [
            145,
            415,
            177,
            433
        ],
        "word_1_58": [
            183,
            419,
            256,
            433
        ],
        "word_1_59": [
            262,
            415,
            353,
            434
        ],
        "word_1_6": [
            379,
            101,
            448,
            119
        ],
        "word_1_60": [
            84,
            443,
            227,
            462
        ],
        "word_1_61": [
            232,
            447,
            359,
            462
        ],
        "word_1_62": [
            365,
            443,
            397,
            462
        ],
        "word_1_63": [
            402,
            443,
            458,
            462
        ],
        "word_1_64": [
            84,
            472,
            148,
            496
        ],
        "word_1_65": [
            154,
            472,
            216,
            490
        ],
        "word_1_66": [
            222,
            473,
            237,
            490
        ],
        "word_1_67": [
            244,
            472,
            291,
            490
        ],
        "word_1_68": [
            297,
            476,
            316,
            490
        ],
        "word_1_69": [
            323,
            472,
            419,
            490
        ],
        "word_1_7": [
            84,
            133,
            103,
            147
        ],
        "word_1_70": [
            425,
            472,
            456,
            490
        ],
        "word_1_71": [
            84,
            501,
            158,
            524
        ],
        "word_1_72": [
            164,
            501,
            240,
            524
        ],
        "word_1_73": [
            246,
            501,
            343,
            524
        ],
        "word_1_74": [
            349,
            501,
            374,
            524
        ],
        "word_1_75": [
            379,
            507,
            389,
            519
        ],
        "word_1_76": [
            84,
            530,
            166,
            553
        ],
        "word_1_77": [
            172,
            529,
            242,
            547
        ],
        "word_1_78": [
            248,
            529,
            291,
            547
        ],
        "word_1_79": [
            298,
            529,
            363,
            547
        ],
        "word_1_8": [
            111,
            129,
            195,
            153
        ],
        "word_1_80": [
            368,
            530,
            383,
            547
        ],
        "word_1_81": [
            84,
            559,
            174,
            581
        ],
        "word_1_82": [
            180,
            559,
            199,
            576
        ],
        "word_1_83": [
            205,
            564,
            216,
            576
        ],
        "word_1_84": [
            223,
            561,
            284,
            576
        ],
        "word_1_85": [
            289,
            564,
            334,
            576
        ],
        "word_1_86": [
            84,
            586,
            222,
            610
        ],
        "word_1_87": [
            226,
            586,
            271,
            604
        ],
        "word_1_88": [
            277,
            586,
            318,
            604
        ],
        "word_1_89": [
            324,
            590,
            344,
            604
        ],
        "word_1_9": [
            200,
            136,
            210,
            147
        ],
        "word_1_90": [
            349,
            586,
            380,
            604
        ],
        "word_1_91": [
            84,
            616,
            210,
            638
        ],
        "word_1_92": [
            217,
            615,
            308,
            633
        ],
        "word_1_93": [
            313,
            616,
            338,
            633
        ],
        "word_1_94": [
            345,
            621,
            355,
            633
        ],
        "word_1_95": [
            361,
            615,
            424,
            634
        ],
        "word_1_96": [
            432,
            621,
            442,
            633
        ],
        "word_1_97": [
            85,
            643,
            175,
            662
        ],
        "word_1_98": [
            180,
            650,
            229,
            667
        ],
        "word_1_99": [
            234,
            643,
            255,
            661
        ]
    },
    "ocr_text": {
        "block_1_1": "|\nIn facsimilea photocell is caused\nto perform a raster scan over the\nsubject copy. The variations of print\ndensity on the document cause the\nphotocell to generate an analogous\nelectrical video signal. This signal is\nused to modulate a carrier, which is\ntransmitted to a remote destination\nover a radio or cable\ncommunications link.",
        "block_1_2": "At the remote terminal.\ndemodulation reconstructs the video\nsignal, which is used to modulate the\ndensity ofprint produced by a\nprinting device. This device is\nscanning in a raster scan\nsynchronised with that at the\ntransmitting terminal. As a result, a\nfacsimile copy of the subject\ndocument is produced.",
        "block_1_3": "@ Wei/PUP com",
        "document": "|\nIn facsimilea photocell is caused\nto perform a raster scan over the\nsubject copy. The variations of print\ndensity on the document cause the\nphotocell to generate an analogous\nelectrical video signal. This signal is\nused to modulate a carrier, which is\ntransmitted to a remote destination\nover a radio or cable\ncommunications link.\n\nAt the remote terminal.\ndemodulation reconstructs the video\nsignal, which is used to modulate the\ndensity ofprint produced by a\nprinting device. This device is\nscanning in a raster scan\nsynchronised with that at the\ntransmitting terminal. As a result, a\nfacsimile copy of the subject\ndocument is produced.\n\n@ Wei/PUP com",
        "line_1_1": "|",
        "line_1_10": "over a radio or cable",
        "line_1_11": "communications link.",
        "line_1_12": "At the remote terminal.",
        "line_1_13": "demodulation reconstructs the video",
        "line_1_14": "signal, which is used to modulate the",
        "line_1_15": "density ofprint produced by a",
        "line_1_16": "printing device. This device is",
        "line_1_17": "scanning in a raster scan",
        "line_1_18": "synchronised with that at the",
        "line_1_19": "transmitting terminal. As a result, a",
        "line_1_2": "In facsimilea photocell is caused",
        "line_1_20": "facsimile copy of the subject",
        "line_1_21": "document is produced.",
        "line_1_22": "@ Wei/PUP com",
        "line_1_3": "to perform a raster scan over the",
        "line_1_4": "subject copy. The variations of print",
        "line_1_5": "density on the document cause the",
        "line_1_6": "photocell to generate an analogous",
        "line_1_7": "electrical video signal. This signal is",
        "line_1_8": "used to modulate a carrier, which is",
        "line_1_9": "transmitted to a remote destination",
        "page_1": "|\nIn facsimilea photocell is caused\nto perform a raster scan over the\nsubject copy. The variations of print\ndensity on the document cause the\nphotocell to generate an analogous\nelectrical video signal. This signal is\nused to modulate a carrier, which is\ntransmitted to a remote destination\nover a radio or cable\ncommunications link.\n\nAt the remote terminal.\ndemodulation reconstructs the video\nsignal, which is used to modulate the\ndensity ofprint produced by a\nprinting device. This device is\nscanning in a raster scan\nsynchronised with that at the\ntransmitting terminal. As a result, a\nfacsimile copy of the subject\ndocument is produced.\n\n@ Wei/PUP com",
        "par_1_1": "|",
        "par_1_2": "In facsimilea photocell is caused\nto perform a raster scan over the\nsubject copy. The variations of print\ndensity on the document cause the\nphotocell to generate an analogous\nelectrical video signal. This signal is\nused to modulate a carrier, which is\ntransmitted to a remote destination\nover a radio or cable\ncommunications link.",
        "par_1_3": "At the remote terminal.\ndemodulation reconstructs the video\nsignal, which is used to modulate the\ndensity ofprint produced by a\nprinting device. This device is\nscanning in a raster scan\nsynchronised with that at the\ntransmitting terminal. As a result, a\nfacsimile copy of the subject\ndocument is produced.",
        "par_1_4": "@ Wei/PUP com",
        "word_1_1": "|",
        "word_1_10": "raster",
        "word_1_100": "the",
        "word_1_101": "subject",
        "word_1_102": "document",
        "word_1_103": "is",
        "word_1_104": "produced.",
        "word_1_105": "@",
        "word_1_106": "Wei/PUP",
        "word_1_107": "com",
        "word_1_11": "scan",
        "word_1_12": "over",
        "word_1_13": "the",
        "word_1_14": "subject",
        "word_1_15": "copy.",
        "word_1_16": "The",
        "word_1_17": "variations",
        "word_1_18": "of",
        "word_1_19": "print",
        "word_1_2": "In",
        "word_1_20": "density",
        "word_1_21": "on",
        "word_1_22": "the",
        "word_1_23": "document",
        "word_1_24": "cause",
        "word_1_25": "the",
        "word_1_26": "photocell",
        "word_1_27": "to",
        "word_1_28": "generate",
        "word_1_29": "an",
        "word_1_3": "facsimilea",
        "word_1_30": "analogous",
        "word_1_31": "electrical",
        "word_1_32": "video",
        "word_1_33": "signal.",
        "word_1_34": "This",
        "word_1_35": "signal",
        "word_1_36": "is",
        "word_1_37": "used",
        "word_1_38": "to",
        "word_1_39": "modulate",
        "word_1_4": "photocell",
        "word_1_40": "a",
        "word_1_41": "carrier,",
        "word_1_42": "which",
        "word_1_43": "is",
        "word_1_44": "transmitted",
        "word_1_45": "to",
        "word_1_46": "a",
        "word_1_47": "remote",
        "word_1_48": "destination",
        "word_1_49": "over",
        "word_1_5": "is",
        "word_1_50": "a",
        "word_1_51": "radio",
        "word_1_52": "or",
        "word_1_53": "cable",
        "word_1_54": "communications",
        "word_1_55": "link.",
        "word_1_56": "At",
        "word_1_57": "the",
        "word_1_58": "remote",
        "word_1_59": "terminal.",
        "word_1_6": "caused",
        "word_1_60": "demodulation",
        "word_1_61": "reconstructs",
        "word_1_62": "the",
        "word_1_63": "video",
        "word_1_64": "signal,",
        "word_1_65": "which",
        "word_1_66": "is",
        "word_1_67": "used",
        "word_1_68": "to",
        "word_1_69": "modulate",
        "word_1_7": "to",
        "word_1_70": "the",
        "word_1_71": "density",
        "word_1_72": "ofprint",
        "word_1_73": "produced",
        "word_1_74": "by",
        "word_1_75": "a",
        "word_1_76": "printing",
        "word_1_77": "device.",
        "word_1_78": "This",
        "word_1_79": "device",
        "word_1_8": "perform",
        "word_1_80": "is",
        "word_1_81": "scanning",
        "word_1_82": "in",
        "word_1_83": "a",
        "word_1_84": "raster",
        "word_1_85": "scan",
        "word_1_86": "synchronised",
        "word_1_87": "with",
        "word_1_88": "that",
        "word_1_89": "at",
        "word_1_9": "a",
        "word_1_90": "the",
        "word_1_91": "transmitting",
        "word_1_92": "terminal.",
        "word_1_93": "As",
        "word_1_94": "a",
        "word_1_95": "result,",
        "word_1_96": "a",
        "word_1_97": "facsimile",
        "word_1_98": "copy",
        "word_1_99": "of"
    },
    "properties": {
        "block_1_1": [
            "bbox 83 68 449 376"
        ],
        "block_1_2": [
            "bbox 84 415 458 696"
        ],
        "block_1_3": [
            "bbox 426 754 536 769"
        ],
        "line_1_1": [
            "bbox 83 68 86 96",
            "baseline 0 0"
        ],
        "line_1_10": [
            "bbox 84 329 292 347",
            "baseline 0 0"
        ],
        "line_1_11": [
            "bbox 85 358 300 376",
            "baseline 0 0"
        ],
        "line_1_12": [
            "bbox 117 415 353 434",
            "baseline 0 -1"
        ],
        "line_1_13": [
            "bbox 84 443 458 462",
            "baseline 0 0"
        ],
        "line_1_14": [
            "bbox 84 472 456 496",
            "baseline 0 -6"
        ],
        "line_1_15": [
            "bbox 84 501 389 524",
            "baseline 0 -5"
        ],
        "line_1_16": [
            "bbox 84 529 383 553",
            "baseline 0 -6"
        ],
        "line_1_17": [
            "bbox 84 559 334 581",
            "baseline 0 -5"
        ],
        "line_1_18": [
            "bbox 84 586 380 610",
            "baseline 0 -6"
        ],
        "line_1_19": [
            "bbox 84 615 442 638",
            "baseline 0 -5"
        ],
        "line_1_2": [
            "bbox 118 101 448 124",
            "baseline 0 -5"
        ],
        "line_1_20": [
            "bbox 85 643 371 667",
            "baseline 0 -6"
        ],
        "line_1_21": [
            "bbox 84 672 316 696",
            "baseline 0 -6"
        ],
        "line_1_22": [
            "bbox 426 754 536 769",
            "baseline 0 -3"
        ],
        "line_1_3": [
            "bbox 84 129 416 153",
            "baseline 0 -6"
        ],
        "line_1_4": [
            "bbox 84 158 449 182",
            "baseline 0 -6"
        ],
        "line_1_5": [
            "bbox 84 186 435 210",
            "baseline 0 -5"
        ],
        "line_1_6": [
            "bbox 84 215 439 239",
            "baseline 0 -6"
        ],
        "line_1_7": [
            "bbox 85 244 444 267",
            "baseline 0 -5"
        ],
        "line_1_8": [
            "bbox 84 272 444 292",
            "baseline 0 -2"
        ],
        "line_1_9": [
            "bbox 84 301 446 319",
            "baseline 0 0"
        ],
        "page_1": [
            "image \"clip_image0029.jpg\"",
            "bbox 0 0 545 771",
            "ppageno 0"
        ],
        "par_1_1": [
            "bbox 83 68 86 96"
        ],
        "par_1_2": [
            "bbox 84 101 449 376"
        ],
        "par_1_3": [
            "bbox 84 415 458 696"
        ],
        "par_1_4": [
            "bbox 426 754 536 769"
        ],
        "word_1_1": [
            "bbox 83 68 86 96",
            "x_wconf 98"
        ],
        "word_1_10": [
            "bbox 216 133 277 147",
            "x_wconf 83"
        ],
        "word_1_100": [
            "bbox 259 643 290 661",
            "x_wconf 87"
        ],
        "word_1_101": [
            "bbox 298 643 371 667",
            "x_wconf 85"
        ],
        "word_1_102": [
            "bbox 84 672 187 690",
            "x_wconf 86"
        ],
        "word_1_103": [
            "bbox 192 673 207 690",
            "x_wconf 89"
        ],
        "word_1_104": [
            "bbox 214 672 316 696",
            "x_wconf 81"
        ],
        "word_1_105": [
            "bbox 426 754 438 766",
            "x_wconf 75"
        ],
        "word_1_106": [
            "bbox 443 754 502 769",
            "x_wconf 50"
        ],
        "word_1_107": [
            "bbox 508 757 536 766",
            "x_wconf 72"
        ],
        "word_1_11": [
            "bbox 283 136 328 147",
            "x_wconf 86"
        ],
        "word_1_12": [
            "bbox 334 136 380 147",
            "x_wconf 89"
        ],
        "word_1_13": [
            "bbox 385 129 416 147",
            "x_wconf 89"
        ],
        "word_1_14": [
            "bbox 84 158 158 181",
            "x_wconf 82"
        ],
        "word_1_15": [
            "bbox 162 164 215 182",
            "x_wconf 83"
        ],
        "word_1_16": [
            "bbox 220 158 258 176",
            "x_wconf 88"
        ],
        "word_1_17": [
            "bbox 265 159 368 176",
            "x_wconf 69"
        ],
        "word_1_18": [
            "bbox 374 158 395 176",
            "x_wconf 84"
        ],
        "word_1_19": [
            "bbox 399 159 449 182",
            "x_wconf 83"
        ],
        "word_1_2": [
            "bbox 118 102 137 119",
            "x_wconf 89"
        ],
        "word_1_20": [
            "bbox 84 186 158 210",
            "x_wconf 84"
        ],
        "word_1_21": [
            "bbox 164 193 190 205",
            "x_wconf 88"
        ],
        "word_1_22": [
            "bbox 195 186 226 205",
            "x_wconf 93"
        ],
        "word_1_23": [
            "bbox 234 186 337 205",
            "x_wconf 81"
        ],
        "word_1_24": [
            "bbox 342 193 398 205",
            "x_wconf 80"
        ],
        "word_1_25": [
            "bbox 404 186 435 205",
            "x_wconf 85"
        ],
        "word_1_26": [
            "bbox 84 215 179 239",
            "x_wconf 79"
        ],
        "word_1_27": [
            "bbox 184 219 203 233",
            "x_wconf 88"
        ],
        "word_1_28": [
            "bbox 211 219 300 239",
            "x_wconf 75"
        ],
        "word_1_29": [
            "bbox 305 222 329 233",
            "x_wconf 94"
        ],
        "word_1_3": [
            "bbox 144 101 249 119",
            "x_wconf 81"
        ],
        "word_1_30": [
            "bbox 335 215 439 239",
            "x_wconf 83"
        ],
        "word_1_31": [
            "bbox 85 244 178 262",
            "x_wconf 81"
        ],
        "word_1_32": [
            "bbox 183 244 238 262",
            "x_wconf 87"
        ],
        "word_1_33": [
            "bbox 245 244 308 267",
            "x_wconf 82"
        ],
        "word_1_34": [
            "bbox 314 244 357 262",
            "x_wconf 88"
        ],
        "word_1_35": [
            "bbox 364 244 422 267",
            "x_wconf 82"
        ],
        "word_1_36": [
            "bbox 430 245 444 262",
            "x_wconf 87"
        ],
        "word_1_37": [
            "bbox 84 272 131 290",
            "x_wconf 88"
        ],
        "word_1_38": [
            "bbox 138 276 157 290",
            "x_wconf 85"
        ],
        "word_1_39": [
            "bbox 163 272 259 290",
            "x_wconf 84"
        ],
        "word_1_4": [
            "bbox 257 101 352 124",
            "x_wconf 88"
        ],
        "word_1_40": [
            "bbox 266 279 276 290",
            "x_wconf 86"
        ],
        "word_1_41": [
            "bbox 282 273 357 292",
            "x_wconf 85"
        ],
        "word_1_42": [
            "bbox 362 272 424 290",
            "x_wconf 92"
        ],
        "word_1_43": [
            "bbox 429 273 444 290",
            "x_wconf 91"
        ],
        "word_1_44": [
            "bbox 84 301 203 319",
            "x_wconf 83"
        ],
        "word_1_45": [
            "bbox 209 304 228 319",
            "x_wconf 88"
        ],
        "word_1_46": [
            "bbox 236 307 246 319",
            "x_wconf 90"
        ],
        "word_1_47": [
            "bbox 253 304 325 319",
            "x_wconf 86"
        ],
        "word_1_48": [
            "bbox 332 301 446 319",
            "x_wconf 81"
        ],
        "word_1_49": [
            "bbox 84 336 131 347",
            "x_wconf 90"
        ],
        "word_1_5": [
            "bbox 357 102 372 119",
            "x_wconf 91"
        ],
        "word_1_50": [
            "bbox 136 336 146 347",
            "x_wconf 96"
        ],
        "word_1_51": [
            "bbox 153 329 205 347",
            "x_wconf 87"
        ],
        "word_1_52": [
            "bbox 212 336 234 347",
            "x_wconf 87"
        ],
        "word_1_53": [
            "bbox 240 329 292 347",
            "x_wconf 82"
        ],
        "word_1_54": [
            "bbox 85 359 253 376",
            "x_wconf 76"
        ],
        "word_1_55": [
            "bbox 258 358 300 376",
            "x_wconf 83"
        ],
        "word_1_56": [
            "bbox 117 416 140 433",
            "x_wconf 91"
        ],
        "word_1_57": [
            "bbox 145 415 177 433",
            "x_wconf 79"
        ],
        "word_1_58": [
            "bbox 183 419 256 433",
            "x_wconf 78"
        ],
        "word_1_59": [
            "bbox 262 415 353 434",
            "x_wconf 84"
        ],
        "word_1_6": [
            "bbox 379 101 448 119",
            "x_wconf 81"
        ],
        "word_1_60": [
            "bbox 84 443 227 462",
            "x_wconf 82"
        ],
        "word_1_61": [
            "bbox 232 447 359 462",
            "x_wconf 67"
        ],
        "word_1_62": [
            "bbox 365 443 397 462",
            "x_wconf 85"
        ],
        "word_1_63": [
            "bbox 402 443 458 462",
            "x_wconf 89"
        ],
        "word_1_64": [
            "bbox 84 472 148 496",
            "x_wconf 79"
        ],
        "word_1_65": [
            "bbox 154 472 216 490",
            "x_wconf 91"
        ],
        "word_1_66": [
            "bbox 222 473 237 490",
            "x_wconf 88"
        ],
        "word_1_67": [
            "bbox 244 472 291 490",
            "x_wconf 89"
        ],
        "word_1_68": [
            "bbox 297 476 316 490",
            "x_wconf 80"
        ],
        "word_1_69": [
            "bbox 323 472 419 490",
            "x_wconf 80"
        ],
        "word_1_7": [
            "bbox 84 133 103 147",
            "x_wconf 87"
        ],
        "word_1_70": [
            "bbox 425 472 456 490",
            "x_wconf 89"
        ],
        "word_1_71": [
            "bbox 84 501 158 524",
            "x_wconf 89"
        ],
        "word_1_72": [
            "bbox 164 501 240 524",
            "x_wconf 83"
        ],
        "word_1_73": [
            "bbox 246 501 343 524",
            "x_wconf 82"
        ],
        "word_1_74": [
            "bbox 349 501 374 524",
            "x_wconf 90"
        ],
        "word_1_75": [
            "bbox 379 507 389 519",
            "x_wconf 92"
        ],
        "word_1_76": [
            "bbox 84 530 166 553",
            "x_wconf 82"
        ],
        "word_1_77": [
            "bbox 172 529 242 547",
            "x_wconf 85"
        ],
        "word_1_78": [
            "bbox 248 529 291 547",
            "x_wconf 87"
        ],
        "word_1_79": [
            "bbox 298 529 363 547",
            "x_wconf 87"
        ],
        "word_1_8": [
            "bbox 111 129 195 153",
            "x_wconf 82"
        ],
        "word_1_80": [
            "bbox 368 530 383 547",
            "x_wconf 89"
        ],
        "word_1_81": [
            "bbox 84 559 174 581",
            "x_wconf 83"
        ],
        "word_1_82": [
            "bbox 180 559 199 576",
            "x_wconf 92"
        ],
        "word_1_83": [
            "bbox 205 564 216 576",
            "x_wconf 90"
        ],
        "word_1_84": [
            "bbox 223 561 284 576",
            "x_wconf 81"
        ],
        "word_1_85": [
            "bbox 289 564 334 576",
            "x_wconf 84"
        ],
        "word_1_86": [
            "bbox 84 586 222 610",
            "x_wconf 63"
        ],
        "word_1_87": [
            "bbox 226 586 271 604",
            "x_wconf 86"
        ],
        "word_1_88": [
            "bbox 277 586 318 604",
            "x_wconf 86"
        ],
        "word_1_89": [
            "bbox 324 590 344 604",
            "x_wconf 90"
        ],
        "word_1_9": [
            "bbox 200 136 210 147",
            "x_wconf 95"
        ],
        "word_1_90": [
            "bbox 349 586 380 604",
            "x_wconf 88"
        ],
        "word_1_91": [
            "bbox 84 616 210 638",
            "x_wconf 73"
        ],
        "word_1_92": [
            "bbox 217 615 308 633",
            "x_wconf 82"
        ],
        "word_1_93": [
            "bbox 313 616 338 633",
            "x_wconf 94"
        ],
        "word_1_94": [
            "bbox 345 621 355 633",
            "x_wconf 90"
        ],
        "word_1_95": [
            "bbox 361 615 424 634",
            "x_wconf 77"
        ],
        "word_1_96": [
            "bbox 432 621 442 633",
            "x_wconf 96"
        ],
        "word_1_97": [
            "bbox 85 643 175 662",
            "x_wconf 80"
        ],
        "word_1_98": [
            "bbox 180 650 229 667",
            "x_wconf 89"
        ],
        "word_1_99": [
            "bbox 234 643 255 661",
            "x_wconf 82"
        ]
    }
}